README.md

```diff
@@ -66,7 +66,7 @@ Questions can be asked by raising an `Issue`.
 - Commit the changes
 
 ```
-  git commit -m "Initial commit"
+  git commit -m "Add-your-github-username"
```

Comment:
lgtm, however I'm not sure if we should directly write the text(ppl might just copy-paste, perhaps having some `<>` might encourage people to use the proper format?)
Perhaps something like- 
```suggestion
  git commit -m "Add <your-github-username>"
```